charts/intel-gpu-resource-driver/templates/resource-driver.yaml

```diff
@@ -70,11 +48,12 @@ spec:
           mountPath: /etc/cdi
         - name: varruncdi
           mountPath: /var/run/cdi
-        - name: defaults
-          mountPath: "/defaults"
-          readOnly: true
+        # when using fake sysfs - mount at the same place as on host
```

Comment:
fake sysfs is not supported this way, it shouldn't be in Helm chart. Later on we can have init container that creates fake sysfs that is used by the main container, but this init container is not yet in ready.